package.json

```diff
@@ -51,7 +51,7 @@
   "dependencies": {
     "@paypal/sdk-constants": "^1.0.61",
     "beaver-logger": "^4.0.0",
-    "belter": "^1.0.16",
+    "belter": "^1.0.136",
```

Comment:
The latest version of belter is `1.0.146`. Should we use that instead? Happy to update it.

I went with 1.0.136 since that's when `ATTRIBUTES.UID` was added.

Reply:
I changed my mind and went with the latest belter `1.0.146`. My thinking is it's a util library so it's always best to have the latest.